### PR TITLE
Add full support for TRV602

### PR DESCRIPTION
**Pull request by lompi**
This pull request adds full support for the TRV602 device. Until now, support for this device was added using the existing configuration, but this device does not support boost mode and gives the possibility to control screen orientation and display brightness.

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -3240,6 +3239,78 @@ const definitions: Definition[] = [
...
+                .withRunningState(['idle', 'heat'], ea.STATE),
+            ...tuya.exposes.scheduleAllDays(ea.STATE_SET, 'HH:MM/C HH:MM/C HH:MM/C HH:MM/C'),
+            e.enum('display_brightness', ea.STATE_SET, ['high', 'medium', 'low']).withDescription('Display brightness'),
+            e.enum('screen_orientation', ea.STATE_SET, ['up', 'right', 'down', 'left']).withDescription('Screen orientation'),
+        ],
+        meta: {
+            tuyaDatapoints: [
+                [1, null,
+                    {
+                        from: (v) => {
```
Comment (Koenkk) [inline src/devices/tuya.ts:3277]: Could you move this to a value converter in `lib/tuya.ts` and also use it for `TS0601_thermostat_1`?

Comment (lompi) [reply src/devices/tuya.ts]: Ok. I moved that. I hope that it's ok, because js isn't my technology and I did it by feel.

**Comment by Koenkk** [comment]
Thanks!

**Comment by BAUBLITZ** [comment]
the TRV602 has an additional function that is still missing in tuya.js.
The “eco” and “comfort” functions. In "eco" the thermostat regulates the temperature by completely opening and closing the valve. With the "comfort" function, the thermostat regulates the temperature much more precisely because the valve is opened or closed to different extents depending on the difference between the set and the actual temperature.

To do this, the following must be added under exposes:

e.enum('mode', ea.STATE_SET, ['comfort', 'eco']).withDescription('Hysteresis - comfort > switches off/on exactly at reached temperature with valve smooth from 0 to 100%, eco > 0.5 degrees above or below, valve either 0 or 100%'),

an under tuyaDatapoints:

[114, 'mode', tuya.valueConverterBasic.lookup({'comfort': tuya.enum(0), 'eco': tuya.enum(1)})],

Can someone still integrate this?

**Comment by Koenkk** [comment]
@BAUBLITZ done!

**Pull request merged**